queue.lua

```diff
@@ -25,8 +25,11 @@ function Qless.queue(name)
         return redis.call('zrem', queue:prefix('work'), unpack(arg))
       end
     end, add = function(now, priority, jid)
+      if priority ~= '+inf' then
+        priority = priority - (now / 10000000000)
```

Comment:
I'm not sure why this if is necessary. `+inf` minus any finite number is `+inf`.